### PR TITLE
docs(configuration): add example for loader option

### PR DESCRIPTION
**Pull request by chenxsan**

### CODE REVIEW

**Review by montogeek — APPROVED** [review]

File: src/content/configuration/other-options.md [hunk 1]
```diff
@@ -304,10 +304,32 @@ W> `cache.idleTimeoutForInitialStore` is only available when [`cache.store`](#ca
...
+module.exports = {
+  // ...
+  loader: {
+    answer: 42
+  }
+};
+```
+
+Then use `this.answer` to get its value in loader:
+
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md:323]: Maybe prefix it with `custom-loader.js` or smth like that for more clarity?

File: src/content/configuration/other-options.md [hunk 2]
```diff
@@ -304,10 +304,32 @@ W> `cache.idleTimeoutForInitialStore` is only available when [`cache.store`](#ca
...
+```javascript
+module.exports = function (source) {
+  // ...
+  console.log(this.answer); // will log `42` here
+  return source;
+};
+```
+
+T> You can override any properties in loader context as webpack copies all properties defined in `loader` to the loader context.
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md:332]: some grammar suggestion:
```suggestion
T> You can override properties in the loader context as webpack copies all properties that are defined in the `loader` to the loader context.
```

**Review by EugeneHlushko — CHANGES_REQUESTED** [review]

File: src/content/configuration/other-options.md [hunk 3]
```diff
@@ -304,10 +304,34 @@ W> `cache.idleTimeoutForInitialStore` is only available when [`cache.store`](#ca
 
 `object`
 
-Expose custom values into the loader context.
+Expose custom values into the [loader context](/api/loaders/#the-loader-context).
 
-?> Add an example...
+For example, you can define a new variable in loader context:
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md:309]:
```suggestion
For example, you can define a new variable in the loader context:
```

File: src/content/configuration/other-options.md [hunk 4]
```diff
@@ -304,10 +304,34 @@ W> `cache.idleTimeoutForInitialStore` is only available when [`cache.store`](#ca
...
+```javascript
+module.exports = {
+  // ...
+  loader: {
+    answer: 42
+  }
+};
+```
+
+Then use `this.answer` to get its value in loader:
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md:322]:
```suggestion
Then use `this.answer` to get its value in the loader:
```

**Review by EugeneHlushko — CHANGES_REQUESTED** [review]
did i miss them first time or were they added later? sorry if missed

**Review by sokra — APPROVED** [review]

**Comment by sokra** [comment]
Thanks

**Pull request merged**